testing/src/test/java/org/motechproject/nms/testing/it/api/MobileAcademyControllerBundleIT.java

```diff
@@ -158,4 +192,469 @@ public void testSmsStatusInvalidFormat() throws IOException, InterruptedExceptio
         assertTrue(SimpleHttpClient.execHttpRequest(request, HttpStatus.SC_BAD_REQUEST, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD));
     }
 
+    /**
+     * setup MA course structure from nmsCourse.json file.
+     */
+    private JSONObject setupMaCourse() throws IOException {
+        MaCourse course = new MaCourse();
+        String jsonText = IOUtils
```

Comment:
It's very easy to use intelliJ to reformat your code such that the lines are wrapped at the right width (set in **Settings/Editor/Code Style/Right margin**) make sure that the **Line breaks** checkbox is unchecked under **Settings/Editor/Code Style/Java/Keep** and then select the code (or nothing to reformat the entire file) and select **Code/Reformat Code...**